src/validator/CreditCardVendorsReader.java

```diff
@@ -0,0 +1,58 @@
+package validator;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class CreditCardVendorsReader implements CreditCardVendorReading {
```

Comment:
This class should be controverted to factory which provides CeditCardVendor abstract for concrete vendor implantation